admin/schema_changes/16.sql

```diff
@@ -1 +1,2 @@
 ALTER TYPE entity_types ADD VALUE 'artist' AFTER 'place';
+ALTER TYPE entity_types ADD VALUE 'label' AFTER 'artist';
```

Comment:
I just saw this, this should be in a different script so I can run it on the main db. Generally, each PR should have a single schema change script.

Reply:
Ah, right. @spellew Please add schema change scripts to `admin/schema_changes` (One for `artists` and one for `labels`)

Reply:
Thank you @paramsingh for pointing that out :)

Reply:
And for fixing it too.